### PR TITLE
Fix/irdium features fixes

### PR DESCRIPTION
**Pull request by sridevi-15**
#### Fixes #.

- [x] Ran `flake8`
- [x] Ran `run_repo_tests.sh`

#### Changes proposed in this pull request:

- Added arthimetic test to verify the crash for all platforms
- converted float to double in CBL to get the right value

### CODE REVIEW

File: testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_database.py [hunk 1]
```diff
@@ -330,3 +330,29 @@ def test_copy_prebuilt_database(params_from_base_test_setup):
 
     # Cleaning the database , tearing down
     db.deleteDB(cbl_db1)
+
+
+@pytest.mark.listener
+@pytest.mark.replication
+def test_query_arthimetic(params_from_base_test_setup):
```
Comment (hrajput89) [inline testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_database.py]: this test should be added in test_query.py, why did we add here.

File: testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_database.py [hunk 2]
```diff
@@ -330,3 +330,29 @@ def test_copy_prebuilt_database(params_from_base_test_setup):
...
+    """
+        @summary:
+        1. Clean up the database/ Remove any existing database.
```
Comment (hrajput89) [inline testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_database.py]: I think summary is wrong.

File: testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_database.py (hunk 1 above)
Comment (euniceatcouchbase) [reply testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_database.py]: same comment here

Comment (sridevi-15) [reply testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_database.py]: done

File: testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_database.py (hunk 2 above)
Comment (sridevi-15) [reply testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_database.py]: done

File: testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_database.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 from CBLClient.Database import Database
 from CBLClient.DatabaseConfiguration import DatabaseConfiguration
-
+from CBLClient.Query import Query
```
Comment (euniceatcouchbase) [inline testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_database.py]: seems you moved the test case to Query section, therefore this import is no longer needed?

Comment (sridevi-15) [reply testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_database.py]: done. Sorry forgot to run flake8. I ran and fixed flake8 issues.

**Comment by sridevi-15** [comment]
fixed flake8

**Pull request merged**